recipes/ms-gsl/all/conanfile.py

```diff
@@ -1,54 +1,60 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-from conans import ConanFile, tools
 import os
+from conans import ConanFile, tools
 
 
 class GslMicrosoftConan(ConanFile):
     name = "ms-gsl"
-    version = "2.0.0"
-    description = "Functions and types that are suggested for use by the C++ Core Guideline"
-    url = "https://github.com/bincrafters/conan-gsl_microsoft"
+    version = "2.1.0"
```

Comment:
you will need to remove the explicit version variable

Reply:
Done.